wikimedia/wikipedia-ios
# Add additional language variant strings for early translation

**Phabricator:** https://phabricator.wikimedia.org/T273666 (just the adding strings component of this ticket)

### Notes
Apologies - this one's a lot of reading and finicky detail stuff to review. You should only need to review the `CommonStrings.swift` changes, as the rest is autogenerated.

Adds language variant strings for early translation by the community (like https://github.com/wikimedia/wikipedia-ios/pull/3839). I agree with [Toni's comment here](https://phabricator.wikimedia.org/T272416#6838809) that it may be advantageous to have generic strings to use, because as you can see here the original approach for just Chinese doesn't necessarily scale nicely as the languages list grows. But, at least getting this work merged now means we're more likely to have fully translated specialized strings before the feature launches.


### Test Steps
1. Confirm each language from https://phabricator.wikimedia.org/T273666 is represented and the string values match
2. Ensure each localized string's a) variable name b) key c) value and d) description are using the corresponding and appropriate language
3. Ensure there are no errant semicolons (semicolons from the Phab ticket body text should have been replaced with newlines `\n`)
